scripts/ci-deb-build-test

```diff
@@ -19,6 +21,21 @@ apt-get install -y \
     cryptsetup \
     tzdata \
     curl wget git
+
+if [ $OS_MAJOR -gt 23 ];  then 
+# install dependencies for 24.04 (ubuntu 24.04 does not have python2)
```

Comment:
Don't duplicate the whole install command because of one difference.  Take python2 out of the list and only do an if/else for the one package that needs to change.

Reply:
Come to think of it, I don't see python in the debian instructions in either INSTALL.md or dist/debian/DEBIAN_PACKAGE.md.  Do we really use python?  If so, is it only something for the ci check or does it need to be in the general instructions for debian installation and/or packaging?